### DISCUSSION TITLE
GitHub Checks REST API

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32374] [Apps, API and Webhooks]
I have 4 checks in my GitHub Enterprise web page.

Out of these 4 checks, 2 are of Jenkins.

When I am hitting Checks REST API, I am getting on 2 checks in the response which are Github checks i.e. Jenkins ones are not coming in response.

Is there any way to get all of these 4 checks using that REST API?

 

API I am using: `api/v3/repos/{owner}/{repo}/commits/{ref}/check-runs`

**Comment by ivailop** [comment]
If not mistaken, your Jenkins might be setting commit-status (not check-run).
Try using https://docs.github.com/en/rest/commits/statuses API
